src/api/spec/db/data/remove_duplicate_repositories_spec.rb

```diff
@@ -52,7 +52,7 @@
     it 'removes the local and duplicate repository' do
       expect do
         data_migration.up
-      end.to change { Repository.count }.by(-1)
+      end.to change(Repository, :count).by(-1)
```

Comment:
rebase..

Reply:
I cherry-picked that commit. So this shouldn't be a problem. Git will resolve it;-)